scrapers/base.py

```diff
@@ -26,6 +26,8 @@ def __init__(self, teiid_user=None, teiid_password=None, kerberos=False, neo4j_u
         :kwarg neo4j_server: a string of the FQDN of the Neo4j server
         :return: None
         """
+        # Defining URL for Freshmaker API
+        self.freshmaker_url = "https://freshmaker.engineering.redhat.com/api/1/events/?per_page=100"
```

Comment:
This should be on the Freshmaker scraper since other scrapers shouldn't need it. See what I did in the dist-git scraper as an example.